api/HTMLMediaElement.json

```diff
@@ -923,13 +923,13 @@
             "firefox": {
               "version_added": false,
               "notes": [
-                "Firefox doesn't implement this yet. See bug <a href='https://bugzil.la/847377'>847377</a>."
+                "Firefox doesn't implement this yet. See bug <a href='https://bugzil.la/847377'>bug 847377</a>."
```

Comment:
Bug has been doubled up here (i.e., "See bug bug 84377")

Reply:
I have added a check for this (64deac3) and fixed all occurences (2492120):

![image](https://user-images.githubusercontent.com/495429/43284261-88f70fa2-911b-11e8-8020-f0becccb1761.png)